specification/billing/resource-manager/Microsoft.Billing/preview/2018-03-01-preview/billingV2.json

```diff
@@ -0,0 +1,394 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-03-01-preview",
+    "title": "BillingManagementClient",
+    "description": "Billing client provides access to billing resources for Azure subscriptions."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.Billing/enrollmentAccounts": {
+      "get": {
+        "tags": [
+          "EnrollmentAccounts"
+        ],
+        "x-ms-examples": {
+          "EnrollmentAccountsList": {
+            "$ref": "./examples/EnrollmentAccountsList.json"
+          }
+        },
+        "operationId": "EnrollmentAccounts_List",
+        "description": "Lists the enrollment accounts the caller has access to.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/EnrollmentAccountListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
```

Comment:
Can you reference from [common-types](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v1/types.json)? Same for all the other places in this swagger.

Reply:
This is just a subset of the billing.josn so I kept it same. We are planning to deprecate this version after few months. This is just to support the old PowerShell cmdlets in the GA version for time being.